tests/unit/Literal.test.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { ThemeProvider } from 'emotion-theming';
+import { ThemeProvider } from '@emotion/react';
```

Comment:
Looks like unused import. Could we omit it?

Reply:
Awesome catch, thanks!